tools/api-markdown-documenter/pnpm-lock.yaml

```diff
@@ -6535,6 +6535,12 @@ packages:
     resolution: {integrity: sha512-V50KMwwzqJV0NpZIZFwfOD5/lyny3WlSzRiXgA0G7VUnRlqttta1L6UQIHzd6EuBY/cHGfwTIck7w1yH6Q5zUw==}
     dev: true
 
+  file:../../common/build/eslint-config-fluid/src/fluid-internal:
```

Comment:
@Josmithr 

I think these changes with `fluid-internal` are relevant?

Reply:
Oh, right, because `api-markdown-documenter` has a `file:` dependency on the eslint config package. Makes sense.